Doppler.Files.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.0.31903.59
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Doppler.Files.Api", "Doppler.Files.Api\Doppler.Files.Api.csproj", "{4CD0B54A-089B-485B-9739-7DDFEFA6137D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Doppler.Files.Api", "Doppler.Files.Api\Doppler.Files.Api.csproj", "{4CD0B54A-089B-485B-9739-7DDFEFA6137D}"
```

Comment:
Just curiosity, why did this change happen?

Reply:
Checking git history I can see that it happened when unit test project was added, did some research about this issue and causes aren't very clear, it's like the way Visual Studio interpretes the new project, if it's a new or an exisiting one. I'll do some checks with an empty solution, may be related the template I used to create the unit tests project.

Reply:
Thanks! Do not worry. 

If you are interested, it seems to say something: https://stackoverflow.com/questions/10802198/visual-studio-project-type-guids#answer-53485177, I will read it later.

Reply:
Created a solution from scratch, added a web API project, then a xUnit project and always uses "{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}" project type. This seems to be the right project type, that is the original project type of Dopper.Files.API.